feature-libs/product/multi-dimensional/assets/translations/en/productMultiDimensional.json

```diff
@@ -0,0 +1,9 @@
+{
+  "productMultiDimensional": {
+    "Style": "Style",
+    "Size": "Size",
+    "Color": "Color",
+    "Material": "Material",
+    "Fit": "Fit"
+  }
+}
```

Comment:
[MEDIUM]
As we discovered on a call, names of variant categories and variant values can be translated in CMS in backoffice. no need to translate them in Spartacus